src/features/inlay_hints.zig

```diff
@@ -237,6 +237,24 @@ fn writeVariableDeclHint(builder: *Builder, decl_node: Ast.Node.Index) !void {
     );
     if (type_str.len == 0) return;
 
+    // Restrict whitespace to only one space at a time.
+    var reduced_type_str = std.ArrayList(u8).init(builder.arena);
+    // Overallocates by a small amount if whitespace is reduced, but it should be fine.
+    try reduced_type_str.ensureTotalCapacity(type_str.len);
```

Comment:
this can be combined into a call to `std.ArrayListUnmanaged(u8).initCapacity(builder.arena, type_str.len)`